utils/swift_build_support/swift_build_support/products/cmark.py

```diff
@@ -94,8 +94,14 @@ def test(self, host_target):
             # Xcode generator uses "RUN_TESTS" instead of "test".
             results_targets = ['RUN_TESTS']
 
+        test_env = {
+            "CTEST_OUTPUT_ON_FAILURE": "ON"
+        }
+
+        # see the comment in cmake_product.py if you want to copy this code to pass
```

Comment:
Good idea to add a breadcrumb here while keeping the main message in a single place.